src/components/CreateBook/CreateBook.jsx

```diff
@@ -0,0 +1,180 @@
+    language: 'Romanian',
+    author: "test",
+    ageRange: "kids",
+    publishingYear: "1990",
+    status: "open",
+    genre: "Fantasy",
+}
+
+const optionsStatus = [
+    { value: 'open', label: 'Open' },
+    { value: 'borrowed', label: 'Borrowed' },
+];
+
+const optionsAge = [
+    { value: 'kids', label: 'Kids' },
+    { value: 'adults', label: 'Adults' },
+];
+
+const optionsGenre = [
+    { value: 'Fantasy', label: 'Fantasy' },
+    { value: 'Science Fiction', label: 'Science Fiction' },
+    { value: 'Dystopian', label: 'Dystopian' },
+    { value: 'Action & Adventure', label: 'Action & Adventure' },
+    { value: 'Romance', label: 'Romance' },
+    { value: 'Mystery', label: 'Mystery' },
+    { value: 'Horror', label: 'Horror' },
+    { value: 'Thriller', label: 'Thriller' },
+    { value: 'Paranormal', label: 'Paranormal' },
+    { value: 'Western', label: 'Western' },
+    { value: 'Literary Fiction', label: 'Literary Fiction' },
+    { value: 'Historical Fiction', label: 'Historical Fiction' },
+    { value: 'Contemporary Fiction', label: 'Contemporary Fiction' },
+    { value: 'Magic Realism', label: 'Magic Realism' },
+    { value: 'Graphic Novel', label: 'Graphic Novel' },
+    { value: 'Short Story', label: 'Short Story' },
+    { value: 'Young Adult', label: 'Young Adult' },
+    { value: 'New Adult', label: 'New Adult' },
+    { value: 'Biographies', label: 'Biographies' },
+    { value: 'Memoirs & Autobiographies', label: 'Memoirs & Autobiographies' },
+    { value: 'Food & Drink', label: 'Food & Drink' },
+    { value: 'Art & Photography', label: 'Art & Photography' },
+    { value: 'Self-Help & Motivational', label: 'Self-Help & Motivational' },
+    { value: 'History', label: 'History' },
+    { value: 'Crafts, Hobbies & Home', label: 'Crafts, Hobbies & Home' },
+    { value: 'Humor & Entertainment', label: 'Humor & Entertainment' },
+    { value: 'Business & Money', label: 'Business & Money' },
+    { value: 'Law & Criminology', label: 'Law & Criminology' },
+    { value: 'Politics & Social Sciences', label: 'Politics & Social Sciences' },
+    { value: 'Religion & Spirituality', label: 'Religion & Spirituality' },
+    { value: 'Education & Teaching', label: 'Education & Teaching' },
+    { value: 'Travel', label: 'Travel' },
+    { value: 'True Crime', label: 'True Crime' },
+    { value: 'Poetry', label: 'Poetry' },
+    { value: 'Personal Growth', label: 'Personal Growth' },
+];
+
+const CreateBook = ({ bookId }) => {
+
+    const { inputs, handleBulkInput } = useContext(InputContext);
+
+    // placeholder for book update
+    const [testBook, setTestBook] = useState(bookData);
+
+    const [selectedImage, setSelectedImage] = useState(null);
+
+    // console.log(inputs);
+
+    const handleFormSubmit = (event) => {
+        event.preventDefault();
+        return (
+            console.log('this is the submit form')
+        )
+    };
+
+    useEffect(() => {
+        handleBulkInput(testBook);
```

Comment:
When I ran the rpm start, it's saying that it's missing a dependency here, is this for a placeholder?

Reply:
hm... I did not notice this message... only ne one that says that a component is changing an uncontrolled input to be controlled.